File: Android/mobile/src/main/java/com/appgo/appgopro/models/AGNotification.kt
package com.appgo.appgopro.models

import com.google.gson.annotations.SerializedName
import java.util.*

/**
 * Created by KSMA on 4/22/2017.
 */

class AGNotification {

    @SerializedName("per_page")
    var perPage: Int = 0

    @SerializedName("current_page")
    var currentPage: Int = 0

    @SerializedName("next_page_url")
    var nextPageUrl: String? = null

    @SerializedName("prev_page_url")
    var prevPageUrl: String? = null

    @SerializedName("from")
    var fromService: Int = 0

    @SerializedName("to")
    var toService: Int = 0

    @SerializedName("data")
    var notificationDatas: List<NotificationData> = ArrayList()

    inner class NotificationData {

        @SerializedName("id")
        var id: Int = 0

        @SerializedName("title")
        var title: String? = null

        @SerializedName("content")
        var content: String? = null

        @SerializedName("updated_at")
        var updatedAt: String? = null
    }
}

File: windows/Model/AGNotification.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppGo.Model
{
    [Serializable]
    public class AGNotificationData
    {
        [JsonProperty("id")]
        public int id { get; set; } = 0;

        [JsonProperty("title")]
        public string title { get; set; } = "";

        [JsonProperty("content")]
        public string content { get; set; } = "";

        [JsonProperty("updated_at")]
        public string updated_at { get; set; } = "";
    }

    [Serializable]
    public class AGNotification
    {
        [JsonProperty("per_page")]
        public int per_page { get; set; } = 0;

        [JsonProperty("current_page")]
        public int current_page { get; set; } = 0;

        [JsonProperty("next_page_url")]
        public string next_page_url { get; set; } = "";

        [JsonProperty("prev_page_url")]
        public string prev_page_url { get; set; } = "";

        [JsonProperty("from")]
        public string from { get; set; } = "";

        [JsonProperty("to")]
        public string to { get; set; } = "";

        [JsonProperty("data")]
        public List<AGNotificationData> data { get; set; } = new List<AGNotificationData>();
    }
}
